eole/bin/convert/convert_HF.py

```diff
@@ -541,7 +552,6 @@ def run(cls, args):
 
         add_qkvbias = False
         add_ffnbias = False
-        rotary_interleave = False
```

Comment:
where do you set rope_config.rotary_interleave = False for HF models now ?

Reply:
Good catch, last changes in convert_HF are messed up. Will fix.

Reply:
Fixed in 54ae2c2